src/hooks/screen/useCreateChat.ts

```diff
@@ -113,6 +113,7 @@ const useCreateChat = () => {
       const initChannel = await SignedMessageRepo.createSignedChat(members);
       const chatData = createChannelJson(initChannel, selectedUser);
 
+      console.log('initChannel', JSON.stringify(initChannel));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive data like the initial channel information can pose a security risk. If this log is for debugging purposes, it should be removed before merging to production.
```diff
- console.log('initChannel', JSON.stringify(initChannel));
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->